it/controllers/IndexControllerISpec.scala

```diff
@@ -542,14 +967,69 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     latePaymentPenalties = Some(Seq.empty[LatePaymentPenalty])
   )
 
+  val getPenaltiesDetailsPayloadWithMultiplePenaltyPeriodInLSP: GetPenaltyDetails = GetPenaltyDetails(
+    totalisations = Some(Totalisations(
+      LSPTotalValue = Some(200),
+      penalisedPrincipalTotal = Some(2000),
+      LPPPostedTotal = Some(165.25),
+      LPPEstimatedTotal = Some(15.26),
+      LPIPostedTotal = Some(1968.2),
+      LPIEstimatedTotal = Some(7)
```

Comment:
These can be set to None

Reply:
Done, set totalisations = None (review above)